ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/EthAccountsIntegrationTest.java

```diff
@@ -21,14 +21,14 @@
 
 import io.netty.handler.codec.http.HttpHeaderValues;
 import io.vertx.core.json.Json;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 import org.web3j.protocol.core.Request;
 import org.web3j.protocol.core.methods.response.EthAccounts;
 
-public class EthAccountsIntegrationTest extends IntegrationTestBase {
+class EthAccountsIntegrationTest extends IntegrationTestBase {
```

Comment:
is removing public/protected keywords needed for this change?

Reply:
In Junit5 this is the accepted practise.

Reply:
It's not a required change. But with Junit5 it's no longer required to use public scope and it removes warnings about unnecessary scope.